packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -27,6 +27,7 @@ import { MissingEnvUserError } from "./error/missingEnvError";
 
 const actionName = "aadApp/update"; // DO NOT MODIFY the name
 const helpLink = "https://aka.ms/teamsfx-actions/aadapp-update";
+const ViewAadAppHelpLink = "https://aka.ms/teamsfx-view-aad-app";
```

Comment:
This aka link is an old one, maybe we need a new aka link pointing to new position? Since V2&V3 will exists together for a while.

Reply:
thanks, create a new link https://aka.ms/teamsfx-view-aad-app-v3